completions/man

```diff
@@ -99,12 +99,12 @@ _man()
     fi
 
     # shellcheck disable=SC2053
-    if [[ $prev != $mansect ]]; then
-        # File based completion for the rest, prepending ./ if needed
+    if [[ $prev != $mansect && -z ${COMPREPLY-} ]]; then
```

Comment:
We could instead loop through the completions found before `start` below and check and remove duplicates from file based ones, but that'd be very slow when there are lots of man page completions from man path (e.g. `man <TAB>` gives 9000+ on the system I'm looking at).